using-mobile-extensions/adobe-analytics-edge/adobe-analytics-edge-migration-guide.md

```diff
@@ -0,0 +1,55 @@
+# Migrating to Adobe Analytics on Edge Network in Adobe Experience Platform Launch
+
+This guide illustrates how you may migrate from Adobe Analytics direct to Adobe Analytics on Edge Network. You can find the API documentation for Adobe Analytics on Edge Network [here.](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/blob/AEP-Edge-Docs/using-mobile-extensions/adobe-analytics-edge/analytics-edge-api-reference.md)
+
+## Prerequisites
+
+* Access to Adobe Experience Platform
+* Access to Adobe Experience Launch dashboard
+* Minimal Swift / Android development knowledge 
+* General knowledge about the Adobe Experience Platform Mobile SDKs
+
+## Migrate to Adobe Analytics on Edge Network in Experience Platform Launch
+
+### Install Adobe Analytics on Edge Network Extension in Experience Platform Launch
+
+1. In Experience Platform Launch, click **New Property**. Type a name for the property and select **Mobile** as the platform. Click **Save** to create the mobile property.
+2. , In your newly created mobile property, click the **Extensions** tab and [configure the Adobe Experience Platform Edge extension.](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/AEP-Edge-Docs/using-mobile-extensions/adobe-analytics-edge#configure-the-adobe-experience-platform-edge-extension-in-experience-platform-launch)
+3. On the **Catalog** tab, locate or search for the **Adobe Analytics on Edge Network** extension, and click **Install**.
+
+![](../../.gitbook/assets/adobe-analytics-on-edge-network-launch-card.png)
+
+### Copying existing rules from property that uses Adobe Analytics
+
+1. In your mobile property, which needs to be migrated from Adobe Analytics direct to Adobe Analytics on the Edge Network, click the **Rules** tab.
+2. Select all the rules from your property and copy them to the newly created property.
+
+![](../../.gitbook/assets/copying-rules-1.png)
+
+![](../../.gitbook/assets/copying-rules-2.png)
+
+### Reconfiguring existing rules to use track configuration from Adobe Analytics on Edge Network
+
+1. In your newly created mobile property, click the **Rules** tab.
+2. For each copied rule which has **Adobe Analytics Track action**, 
+    * Select the rule from the rules list.
+    * Select each Adobe Analytics Track action from the **Actions** section in the rule.
+    * Make a note of the **Action**, **State** and **Context Data** values have been entered on the right pane.
```

Comment:
Suggestion:
Write down the values of the **Action**, **State** and **Context Data** which are on the right pane.  It will be used in the next steps.